app/src/main/res/values/styles.xml

```diff
@@ -48,8 +48,9 @@
     <style name="DarkTheme.PopupOverlay" parent="ThemeOverlay.AppCompat.Dark"/>
 
     <style name="DarkThemeContrast" parent="DarkTheme">
-        <item name="android:background">#000000</item>
+        <item name="android:windowBackground">@color/darkContrastBlack</item>
         <item name="android:textColor">#FFFFFF</item>
+        <item name="colorPrimary">#000000</item>
```

Comment:
does it make sense to use @color/darkContrastBlack instead of the hex code?

Reply:
I thought it would be easier to change the colors afterwards when they are stored in a variable. However, I can change the hex codes to the color variables to make everything uniform.